lib/pbench/agent/tool_meister.py

```diff
@@ -306,10 +172,10 @@ def stop(self):
                 f"Tool({self.name}) has an unexpected stop process running"
             )
 
-        # FIXME - before we "stop" a tool, check to see if a
-        # "{tool}/{tool}.pid" file exists.  If it doesn't wait for a second to
-        # show up, if after a second it does not show up, then give up waiting
-        # and just call the stop method.
+        # Before we "stop" a tool, check to see if a "{tool}/{tool}.pid" file
+        # exists.  If it doesn't, wait for a second to show up, if after a
```

Comment:
This isn't really unclear as written, but it'd be more "correct" to say "wait for a second for it to show up".